src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOFalcon.kt

```diff
@@ -74,65 +78,54 @@ class SwerveModuleIOFalcon(
       steeringSensor.velocityFeedforwardToRawUnits(DrivetrainConstants.PID.STEERING_KFF)
-    steeringConfiguration.motionCruiseVelocity =
+    steeringConfiguration.MotionMagic.MotionMagicCruiseVelocity =
       steeringSensor.velocityToRawUnits(DrivetrainConstants.STEERING_VEL_MAX)
-    steeringConfiguration.motionAcceleration =
+    steeringConfiguration.MotionMagic.MotionMagicAcceleration =
       steeringSensor.accelerationToRawUnits(DrivetrainConstants.STEERING_ACCEL_MAX)
-    steeringConfiguration.peakOutputForward = 1.0
-    steeringConfiguration.peakOutputReverse = -1.0
-    steeringConfiguration.supplyCurrLimit.currentLimit =
+    steeringConfiguration.CurrentLimits.SupplyCurrentLimit =
       DrivetrainConstants.STEERING_SUPPLY_CURRENT_LIMIT.inAmperes
-    steeringConfiguration.supplyCurrLimit.enable = true
+    steeringConfiguration.CurrentLimits.SupplyCurrentLimitEnable = true
 
-    steeringFalcon.setNeutralMode(NeutralMode.Brake) // change back to coast maybe?
+    steeringConfiguration.MotorOutput.NeutralMode =
+      NeutralModeValue.Brake // change back to coast maybe?
     steeringFalcon.inverted = true
-    steeringFalcon.configAllSettings(steeringConfiguration)
-    steeringFalcon.configAllowableClosedloopError(
-      0, steeringSensor.positionToRawUnits(DrivetrainConstants.ALLOWED_STEERING_ANGLE_ERROR)
-    )
+    steeringFalcon.configurator.apply(steeringConfiguration)
 
-    driveConfiguration.slot0.kP =
+    driveConfiguration.Slot0.kP =
       driveSensor.proportionalVelocityGainToRawUnits(DrivetrainConstants.PID.DRIVE_KP)
-    driveConfiguration.slot0.kI =
+    driveConfiguration.Slot0.kI =
       driveSensor.integralVelocityGainToRawUnits(DrivetrainConstants.PID.DRIVE_KI)
-    driveConfiguration.slot0.kD =
+    driveConfiguration.Slot0.kD =
       driveSensor.derivativeVelocityGainToRawUnits(DrivetrainConstants.PID.DRIVE_KD)
-    driveConfiguration.slot0.kF = 0.05425
+    driveConfiguration.Slot0.kV = 0.05425
     //      driveSensor.velocityFeedforwardToRawUnits(DrivetrainConstants.PID.DRIVE_KFF)
-    driveConfiguration.supplyCurrLimit.currentLimit =
+    driveConfiguration.CurrentLimits.SupplyCurrentLimit =
       DrivetrainConstants.DRIVE_SUPPLY_CURRENT_LIMIT.inAmperes
-    driveConfiguration.supplyCurrLimit.triggerThresholdCurrent =
+    driveConfiguration.CurrentLimits.SupplyCurrentThreshold =
       DrivetrainConstants.DRIVE_THRESHOLD_CURRENT_LIMIT.inAmperes
-    driveConfiguration.supplyCurrLimit.triggerThresholdTime =
+    driveConfiguration.CurrentLimits.SupplyTimeThreshold =
       DrivetrainConstants.DRIVE_TRIGGER_THRESHOLD_TIME.inSeconds
-    driveConfiguration.supplyCurrLimit.enable = true
-    driveConfiguration.statorCurrLimit.currentLimit =
+    driveConfiguration.CurrentLimits.SupplyCurrentLimitEnable = true
+    driveConfiguration.CurrentLimits.StatorCurrentLimit =
       DrivetrainConstants.DRIVE_STATOR_CURRENT_LIMIT.inAmperes
-    driveConfiguration.statorCurrLimit.triggerThresholdCurrent =
```

Comment:
i didn't see any time or trigger thresholds for stator current? is this stator current limit stuff important since it isn't enabled